<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadyResults</class>
 <widget class="QDialog" name="ReadyResults">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab results</string>
  </property>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>180</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255)</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QLabel" name="label_30">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>5.00-10.00</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_42">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>50.0-75.0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_43">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>25.0-40.0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_44">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>3.0-7.0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_45">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>M(4.9-5.5), F(4.0-5.0))</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_46">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>M(14.0-17.0), F(10.0-16.0)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_47">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>M(80.0-100), F(76.0-96.0)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_48">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>M(27.0-32.0), F(30.0-35.0)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_49">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>150-400</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>101</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255)</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="label_21">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>WBC</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_22">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>NEU%</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_23">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>LYM%</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_24">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>MON%</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_25">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>RBC</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_26">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>HGB</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_28">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>MCV</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_27">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>MCH</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_29">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>PLT</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>741</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Blood Count</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_fill">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(227, 156, 156)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>91</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color:rgb(207, 143, 143)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color:rgb(207, 143, 143)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">color:rgb(207, 143, 143)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">color:rgb(207, 143, 143)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">color:rgb(207, 143, 143)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">color:rgb(207, 143, 143)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">color:rgb(207, 143, 143)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true">color:rgb(207, 143, 143)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="styleSheet">
       <string notr="true">color:rgb(207, 143, 143)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>11</y>
     <width>330</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>PatientID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>View Results</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
